core/consensus/grandpa/impl/grandpa_impl.cpp

```diff
@@ -86,8 +86,8 @@ namespace kagome::consensus::grandpa {
              "Grandpa will be started with round #{}",
              round_state.round_number + 1);
 
-    auto authorities_res =
-        authority_manager_->authorities(round_state.last_finalized_block, true);
+    auto authorities_res = authority_manager_->authorities(
```

Comment:
The last parameter is very obscure when reading invocations of this method, maybe replace it by an enum or two public versions of the method?